### ISSUE TITLE
[Accessibility] [Password Page][Semantics] Muda’s button is identified as a img

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#867]
**Describe the bug**
The "Muda logo" button is identified as an image with no alt attribute. As behaviour, it should role as button for a better accessible user experience.
Ensure  elements have alternate text or a role of none or presentation
more informationLink opens in a new window
Element Location:
.h-10
To solve this problem, you need to fix at least (1) of the following:
Element does not have an alt attribute
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element's default semantics were not overridden with role="none" or role="presentation"
Found:Automatically
Impact:critical
cat.text-alternatives
wcag2a
wcag111
section508
section508.22.a
TTv5
TT7.a
TT7.b
EN-301-549
EN-9.1.1.1
ACT

**To Reproduce**
1. Go to '(https://muda.cambiatus.io/join?redirect=%2Fdashboard)'
2. Click on "I'm already a member" button
3. Run AxeDev Tools

**Expected behavior**
It should be a button with a label like “muda’s home page” or something like it. The screenreader user doesn’t know what is that image purpose.

**Screenshots**
![image](https://github.com/user-attachments/assets/06b959a4-9a6d-43d9-8b62-16b6e47c1a82)

**Cambiatus Env you are using**
Put the URL of the page you are trying to use

**Desktop (please complete the following information):**
 - OS:Windows
- Browser Chrome
- Version 129.0.6668.101 (Versão oficial) 64 bits

**Additional context**

**Comment by ceciliabelem** [comment]
For mapping all defects statistics, I just updated the spreadsheet. Also, there is a graphic to help understand some kinds of bugs and how they can impact users.
[Muda's Report Spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
